style.css

```diff
@@ -1 +1,267 @@
-/*  Add your styling here */
+.main {
+    display: grid;
+    width: 100%;
+    height: auto;
+  
+  }
+  /*section one*/
+  .one{
+    background-color: white;
+    display: grid;
+    grid-template-rows: auto auto;
+  }
+  /*section one-one*/
+  .one-one {
+
+  
+  }
+  
+  .mobile-container {
+  
+    color: white;
+    border-radius: 10px;
+    justify-self: end;
+  }
+  
+  .topnav {
+    overflow: hidden;
+    position: relative;
+  }
+  
+  .topnav #myLinks {
+  display:hidden ;
+  
+  }
+  
+  .topnav a {
+    color: rgb(19, 18, 18);
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+    display: block;
+    
+  }
+  
+  .topnav a.icon {
+    
+    display: block;
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
+  
+  
+  .burger{
+    width: 2rem;
+    height: 2rem;
+    
+  }
+  .logo-cake{
+    width: 4rem;
+    height: 4rem;
+    justify-self:flex-start;
+  }
+  
+  
+  
+  /*section one-two*/
+  .one-two {
+    text-align: right;
+    }
+  
+   
+  
+  /*section two*/
+  .two{
+  background-color: rgb(129, 241, 163);
+  font-size:1.4rem ;
+  padding:1.5rem ;
+  margin-top:1rem ;
+  
+  
+  }
+  /*section three*/
+  .three{
+   
+    background-image:url(/images/unsplash.jpg);
+    height:350px ;
+    background-size: cover;
+    margin-top:1rem ;
+    
+  }
+  /*section four*/
+  .four{
+    display:grid;
+    width:100%;
+    height: 300px;
+    grid-template-columns: auto auto;
+  gap: 0.5em;
+  margin-top:1rem ;
+  
+  }
+  
+  
+  .four-one {
+    background-image: url(/images/candle-cake.jpg);
+    background-size: cover;
+    width:100%;
+    height: 150px;
+  }
+  .four-two {
+    background-image: url(/images/cup-cake.jpg);
+    background-size: cover;
+    width:100%;
+    height: 150px;
+   
+  }
+  
+  .four-three {
+    background-image: url(/images/donuts.jpg);
+    
+    background-size: cover;
+    width:100%;
+    height: 150px;
+  }
+  .four-four {
+    background-image: url(/images/slice-cake.jpg);
+    background-size: cover;
+    width:100%;
+    height: 150px;
+  }
+  /*section five*/
+  .five {
+    background-color: rgb(1, 157, 247);
+    margin-top:1rem ;
+    margin-bottom: 1rem;
+  
+  
+  }
+  .fa {
+    padding: 20px;
+    font-size: 350px;
+    width: 36%;
+    text-align: center;
+    text-decoration: none;
+    border:0.1px solid  ;
+    border-radius:5px;
+  }
+  
+  
+  
+  @media screen and (min-width:550px) {
+    .one-one {
+      display: none;
+    }
+    
+    .one-three {
+      display: flex;
+       padding: 2rem;
+        background-color: rgb(26, 166, 221);
+        justify-content: space-around;
+        padding-left:50% ;
```

Comment:
If your intention is to make them start from the right you could use:
justify-content:flex-end and have on .item -> padding: 0 0.5rem;

this way when you reduce the screen size you won't lose the right part of the screen
<img width="729" alt="Screenshot 2022-11-12 at 14 54 11" src="https://user-images.githubusercontent.com/40305104/201477325-2eb6a6ff-6fc6-43af-bba2-6613c4a69ba6.png">